pkg/volume/cephfs/cephfs.go

```diff
@@ -299,6 +327,90 @@ func (cephfsVolume *cephfs) execMount(mountpoint string) error {
 	return nil
 }
 
+func (cephfsMounter *cephfsMounter) checkFuseMount() bool {
+	execute := cephfsMounter.plugin.host.GetExec(cephfsMounter.plugin.GetPluginName())
+	switch runtime.GOOS {
+	case "linux":
+		retBytes, err := execute.Run("/bin/ls", "/sbin/mount.fuse.ceph")
+		if err == nil && string(retBytes) == "/sbin/mount.fuse.ceph\n" {
+			glog.V(4).Infof("/sbin/mount.fuse.ceph exists, it should be fuse mount")
+			return true
+		}
+		return false
+	}
+	return false
+}
+
+func (cephfsVolume *cephfs) execFuseMount(mountpoint string) error {
+	// cephfs keyring file
+	keyring_file := ""
+	// override secretfile if secret is provided
+	if cephfsVolume.secret != "" {
```

Comment:
add a TODO here. cephfs fuse currently doesn't support secret option. Remove keyring file create once secret option is supported.